controllers/grid/users/author/AuthorGridRow.inc.php

```diff
@@ -94,9 +95,8 @@ public function initialize($request, $template = null)
                     )
                 );
 
-                $authorDao = DAORegistry::getDAO('AuthorDAO'); /** @var AuthorDAO $authorDao */
                 $userDao = DAORegistry::getDAO('UserDAO'); /** @var UserDAO $userDao */
-                $author = $authorDao->getById($rowId);
+                $author = Repo::author()->get($rowId);
```

Comment:
Another place where I'd cast to an `int` just to be safe.

Reply:
Casted